.github/workflows/ui_tests.yml

```diff
@@ -133,9 +133,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        shard: [1, 2, 3, 4, 5]
-        shardTotal: [5]
-    timeout-minutes: 20
+        shard: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

Comment:
Might these cause a disadvantage in terms of server capability?

Reply:
Maybe. But this step used to run on 5 shards x 10minutes = 50mins and there were no issues.

We've now brought it down to 4mins. 10 shards x 4mins = 40mins.

I so I think it's safe

Reply:
Does this cost extra $$$$ ?